### DISCUSSION TITLE
咨询许可证事宜

### DISCUSSION THREAD

**Discussion by RuipingWang1986** [modelscope/ms-agent#434] [Q&A]
尊敬的ModelScope团队：

您好！我在研究中使用了ModelScope-Agent框架，并计划将研究成果整理成一本关于如何利用ModelScope-Agent框架创建智能体的书籍。这本书将包含大量原创内容，包括理论分析、实践案例和代码实现等。在撰写过程中，我注意到ModelScope-Agent的许可证规定，因此想请教您几个问题以确保我的作品符合相关规定。
1. 请问ModelScope-Agent的许可证是否允许我以书籍的形式发表关于该框架的研究成果？
2. 在书籍中，我会详细介绍如何使用ModelScope-Agent创建智能体，并分享一些原创的代码和案例。请问这样是否违反了ModelScope-Agent的许可证规定？
3. 如果可以发表，我需要在书中如何注明ModelScope-Agent的版权信息？是否需要附加其他授权文件？
4. 在书籍出版过程中，我可能需要与出版社分享部分ModelScope-Agent的源代码。请问这会涉及到许可证方面的问题吗？
请您在百忙之中抽空回复，非常感谢您的指导！期待您的回复！
顺祝商祺！

王瑞平
ruiping.wang@ieee.org
2014年5月10日

**Comment by zzhangpurdue** [comment]
ModelScope-Agent遵循Apache-2.0 开源许可：https://github.com/modelscope/modelscope-agent/blob/master/LICENSE <https://github.com/modelscope/modelscope-agent/blob/master/LICENSE >
您可以在提供充分的引用和说明的情况下，使用ModelScope-Agent框架开源代码，无需额外的授权和许可。
感谢您对ModelScope-Agent框架的兴趣。
------------------------------------------------------------------
发件人：RuipingWang1986 ***@***.***>
发送时间：2024年5月10日(星期五) 10:52
***@***.***>
***@***.***>
主　题：[modelscope/modelscope-agent] 咨询许可证事宜 (Discussion #434)
尊敬的ModelScope团队：
您好！我在研究中使用了ModelScope-Agent框架，并计划将研究成果整理成一本关于如何利用ModelScope-Agent框架创建智能体的书籍。这本书将包含大量原创内容，包括理论分析、实践案例和代码实现等。在撰写过程中，我注意到ModelScope-Agent的许可证规定，因此想请教您几个问题以确保我的作品符合相关规定。

 * 
请问ModelScope-Agent的许可证是否允许我以书籍的形式发表关于该框架的研究成果？

 * 
在书籍中，我会详细介绍如何使用ModelScope-Agent创建智能体，并分享一些原创的代码和案例。请问这样是否违反了ModelScope-Agent的许可证规定？

 * 
如果可以发表，我需要在书中如何注明ModelScope-Agent的版权信息？是否需要附加其他授权文件？

 * 
在书籍出版过程中，我可能需要与出版社分享部分ModelScope-Agent的源代码。请问这会涉及到许可证方面的问题吗？
 请您在百忙之中抽空回复，非常感谢您的指导！期待您的回复！
 顺祝商祺！
王瑞平
***@***.*** ***@***.*** >
 2014年5月10日

**Comment by wenmengzhou** [comment]
感谢您对 ModelScope-Agent 项目的关注。ModelScope-Agent（代码地址：[https://github.com/modelscope/modelscope-agent）项目采用](https://github.com/modelscope/modelscope-agent%EF%BC%89%E9%A1%B9%E7%9B%AE%E9%87%87%E7%94%A8) Apache 2.0 协议开源，相对比较宽松友好。在遵守协议要求的前提下，可以放心进行研究、使用。协议要求包括：
1、尊重他人著作权，如引用较多项目代码，请在书籍的明显位置进行说明。
2、引用 ModelScope-Agent 代码，注意保留其中的著作权、许可声明等，不能随意删除或修改（包括其中第三方著作权和许可声明）。
3、除因客观介绍所研究项目的需要，不能使用阿里巴巴、阿里云、通义实验室等名称或品牌进行作品的推荐、宣传、营销等。
4、引用项目代码时，如对代码有修改，需明确声明修改。
5、禁止夸大或恶意贬低。如，客观介绍您的著作与 ModelScope-Agent 的关系，禁止用“官方认证”、“官方指南”等具有误导性词语。
6、禁止其他不实信息。
